### PR TITLE
Clean mempools code to avoid unnecessary snmpget

### PR DESCRIPTION
**Pull request by PipoCanaja**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by murrant** [comment]
Wow, this code is old... Maybe I should target this first as code to port to an object oriented style module.

**Comment by PipoCanaja** [comment]
Makes sense. Simple mempools (when no or little maths is needed) could even be YAMLized ...

File: includes/discovery/mempools/fortiweb.inc.php [hunk 1]
```diff
@@ -1,8 +1,11 @@
 <?php
-echo 'FORTIWEB-MEMORY-POOL: ';
-$usage = str_replace('"', "", snmp_get($device, 'FORTINET-FORTIWEB-MIB::fwSysMemUsage.0', '-OvQ'));
-if (is_numeric($usage)) {
-    // fwSysMemCapacity: Total physical memory (RAM) installed (MB)
-    // Conversion from megabyte (base 10) to byte (base 10): Precision = 1000*1000
-    discover_mempool($valid_mempool, $device, 0, 'fortiweb', 'Physical Memory', '1000000', null, null);
+
+if ($device['os'] == 'fortiweb') {
```
Comment (Jellyfrog) [inline includes/discovery/mempools/fortiweb.inc.php:3]: These checks are not needed anymore right?

Comment (PipoCanaja) [reply includes/discovery/mempools/fortiweb.inc.php:3]: they are. All files are loaded without condition right now. So we end up testing fortiweb on all devices for each discovery :)

Comment (murrant) [reply includes/discovery/mempools/fortiweb.inc.php:3]: yes, the mempools code just loops over all the files... as I said very old code.

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-64-release-changelog-may-2020/12290/1

**Pull request merged**